src/com.unity.git.api/Api/Git/GitClient.cs

```diff
@@ -599,6 +614,34 @@ public ITask<string> DiscardAll(IOutputProcessor<string> processor = null)
                 .Configure(processManager);
         }
 
+        ///<inheritdoc/>
+        public ITask<string> Clean(IList<string> files, IOutputProcessor<string> processor = null)
+        {
+            GitCleanTask last = null;
+            foreach (var batch in files.Spool(5000))
```

Comment:
suggestions : add that number to an internal Constants static class

Reply:
I'll look into that!

Reply:
Done!